### PR TITLE
[VEN-2639]: Oracle for rsETH on Ethereum & sepolia

### PR DESCRIPTION
**Pull request by GitGuru7**
## Description

Resolves #

## Checklist

- [ ] I have updated the documentation to account for the changes in the code.
- [ ] If I added new functionality, I added tests covering it.
- [ ] If I fixed a bug, I added a test preventing this bug from silently reappearing again.
- [ ] My contribution follows [Venus contribution guidelines](docs/CONTRIBUTING.md).

### CODE REVIEW

File: deploy/8-deploy-onejump-oracle.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+      owner: proxyOwnerAddress,
+      proxyContract: "OptimizedTransparentProxy",
+    },
+    skipIfAlreadyDeployed: true,
+  });
+};
+
+func.skip = async () => hre.network.name !== "ethereum" && hre.network.name !== "sepolia";
+
+func.tags = ["OneJumpOracle"];
```
Comment (chechu) [inline deploy/8-deploy-onejump-oracle.ts:55]:
```suggestion
func.tags = ["rsETHOneJumpOracles"];
```

Comment (GitGuru7) [reply deploy/8-deploy-onejump-oracle.ts]: [fc77551](https://github.com/VenusProtocol/oracle/pull/203/commits/fc775516a8989b912b788fad551d236336aafd0a)

File: deploy/8-deploy-onejump-oracle.ts [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+        ? "0xfA0614E5C803E15070d31f7C38d2d430EBe68E47"
+        : "0xA1290d69c65A6Fe4DF752f95823fae25cB99e5A7";
+  }
+
+  console.log(`Timelock (${proxyOwnerAddress})`);
+  const redStoneOracle = await hre.ethers.getContract("RedStoneOracle");
+  const resilientOracle = await hre.ethers.getContract("ResilientOracle");
+  const chainlinkOracle = await hre.ethers.getContract("ChainlinkOracle");
+
+  await deploy("OneJumpOracleMain", {
```
[the remark this answers is not in the document]
Comment (GitGuru7) [reply deploy/8-deploy-onejump-oracle.ts]: [fc77551](https://github.com/VenusProtocol/oracle/pull/203/commits/fc775516a8989b912b788fad551d236336aafd0a)

File: deploy/8-deploy-onejump-oracle.ts
[the remark this answers is not in the document]
Comment (GitGuru7) [reply deploy/8-deploy-onejump-oracle.ts:1]: [fc77551](https://github.com/VenusProtocol/oracle/pull/203/commits/fc775516a8989b912b788fad551d236336aafd0a)

**Review by chechu — COMMENTED** [review]
Sepolia deployment lgtm

**Review by kkirka — APPROVED** [review]

**Review by chechu — APPROVED** [review]

**Pull request merged**